objects/rct2/footpath_railings/rct2.footpath_railings.concrete_green/object.json

```diff
@@ -16,7 +16,7 @@
     ],
     "strings": {
         "name": {
-            "en-GB": "Concrete Railings (green supports)",
+            "en-GB": "Concrete Railings (Dark-Green)",
```

Comment:
```suggestion
            "en-GB": "Concrete Railings (Dark Green)",
```

Reply:
You are correct. Looking it up it seems like all people use Dark Green rather than Dark-Green, I will change it accordingly.